zx.mjs

```diff
@@ -117,7 +117,7 @@ async function importPath(filepath, origin = filepath) {
   if (ext === '.ts') {
     let {dir, name} = parse(filepath)
     let outFile = join(dir, name + '.mjs')
-    await run`npm_config_yes=true npx tsc --target esnext --module esnext --moduleResolution node ${filepath}`
+    await run`npm_config_yes=true npx -p typescript tsc --target esnext --module esnext --moduleResolution node ${filepath}`
```

Comment:
Tsc comes with ts by default?

Reply:
Yes. https://github.com/microsoft/TypeScript/blob/master/package.json#L24

Reply:
Why do we need to specify it here?

Reply:
To load **typescript** pkg and then invoke its **tsc** bin.

Reply:
I think npx will work without it too. Correct?

Reply:
If pkg name does not equal to its bin we have to specify what exactly we'd like to invoke.

Reply:
I got it. https://www.npmjs.com/package/tsc is deprecated.
I assumed we using an old tsc package.